src/commands/aksNavToPortal/aksNavToPortal.ts

```diff
@@ -25,5 +25,5 @@ export default async function aksNavToPortal(
     }
 
     // armid is in the format: /subscriptions/<sub_id>/resourceGroups/<resource_group>/providers/<container_service>/managedClusters/<aks_clustername>
-    vscode.env.openExternal(vscode.Uri.parse(`https://portal.azure.com/#resource${cluster.result.armId}/overview?source=${meta.name}`));
+    vscode.env.openExternal(vscode.Uri.parse(`https://portal.azure.com/#resource${cluster.result.armId}/overview?referrer-source=vscode&context=${meta.name}`));
```

Comment:
As per Sean's email `?referrer_source=XXX&referrer_campaign=XXX` work?.  Please change to referrer_source and referrer_context

Reply:
Cool, so the `source` and `context` changes to `referrer-source` and `referrer-context`